app/src/molecules/UpdateBanner/index.tsx

```diff
@@ -19,6 +21,7 @@ interface UpdateBannerProps {
   setShowBanner: (arg0: boolean) => void
   handleUpdateClick: () => void
   serialNumber: string
+  isTooHot: boolean
```

Comment:
```suggestion
  isTooHot?: boolean
```
Could we make this optional? I think it will help keep UpdateBanner module agnostic.

Reply:
good call!